test/tools/Modules/UnixSocket/UnixSocket.psd1

```diff
@@ -0,0 +1,16 @@
+@{
+    ModuleVersion = '1.0.0'
+    GUID = '86471f04-5b94-4136-a299-caf98464a06b'
+    Author = 'Carlo Toso'
```

Comment:
I looked through other repository module manifests, this looks like the convention:

```suggestion
    Author = 'PowerShell'
```

Reply:
From a quick search I see there are 17 *.psd1 files:
- 11 with `Author = 'PowerShell'`
- 3 with `Author = 'Microsoft Corporation'`
- 3 with `Author = 'Author Name'`
It doesn't seem it's really a convention (I can follow your suggestion anyway if needed)

Reply:
Done